zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2717,6 +2754,42 @@ components:
           description: All the variables visible in the root scope.
           type: object
 
+    VariableRequest:
```

Comment:
🔧 Looking at the other endpoints, `SetVariablesRequest` might be more appropriate. Unfortunately, this collides with the same-name record from the element instance services. But the REST API model should be as consistent as possible (we're not gonna be able to hold this up forever, I fear, but let's try our best).